src/main/java/com/tabnine/general/StaticConfig.java

```diff
@@ -47,14 +49,14 @@ public class StaticConfig {
   public static final String USER_HOME_PATH_PROPERTY = "user.home";
   public static final String REMOTE_BASE_URL_PROPERTY = "TABNINE_REMOTE_BASE_URL";
   public static final String REMOTE_VERSION_URL_PROPERTY = "TABNINE_REMOTE_VERSION_URL";
+  public static final String TABNINE_ENTERPRISE_HOST = "TABNINE_ENTERPRISE_HOST";
```

Comment:
this will always change it, don't we want it only for onprem?

Reply:
this is not a change, its an added property